app/v1/views/politicalparties.py

```diff
@@ -15,98 +15,73 @@ def add_politicalparties():
     #Create a new political party  - POST request
     
     if request.method == "POST":
-        try: 
-            data = request.get_json()
-            
-            name = data['name']
-            logoUrl = data['logoUrl']
-            members = data['members']
-            headquarters = data['headquarters']
-            chairperson = data['chairperson']
-
-            new_politicalparty ={ 
-                    "party_id": len(politicalparties_list) + 1,
-                    "name" : name ,
-                    "logoUrl" : logoUrl ,
-                    "members" : members ,
-                    "headquarters": headquarters,
-                    "chairperson": chairperson
-                    }
-            if not name.isalpha() or not logoUrl.isalpha() or not headquarters.isalpha() or not chairperson.isalpha():
-                pass
-            if not members.isdigit():
-                pass
-            
-            
-
-        except:
-                return response (400, "Please fill in all the fields as required: members as numbers, the rest as text",[])
-
+        data = request.get_json()
+        name = data['name']
+        logoUrl = data['logoUrl']
+        hqAddress = data['hqAddress']
+
+        if not name.isalpha() or not logoUrl.isalpha() or not hqAddress.isalpha():
+            return response (400, "Please fill in all the fields as required: members as numbers, the rest as text",[])
```

Comment:
One may not be aware of what the **rest** means, please list all the inputs required just like you have done for **members**